instrumentation/logback/logback-appender-1.0/javaagent/src/test/groovy/LogbackTest.java

```diff
@@ -0,0 +1,241 @@
+/*
```

Comment:
Should we move the file to `src/test/java` source root?

Reply:
To align with existing java tests I believe LogbackTest should also be moved to `io.opentelemetry.javaagent.instrumentation.logback.appender.v1_0`